doc/org-roam.org

```diff
@@ -646,6 +646,68 @@ Org-roam also provides some functions to add or remove refs.
 
   Remove a ref from the node at point.
 
+* Citations
+
+Since version 9.5, Org has first-class support for citations. Org-roam supports
+the caching of both these in-built citations (of form ~[cite:@key]~) and [[https://github.com/jkitchin/org-ref][org-ref]]
+citations (of form cite:key).
+
+** Setup
+
+*org-ref*
+
+To ensure that org-ref citations are cached, ~org-ref~ must be loaded prior to
+calls of ~org-roam-db-sync~. This can be achieved by placing ~(require
+'org-ref)~ before ~org-roam~ is loaded, for example in the ~:init~ block of the
+Org-roam use-package declaration:
+
+#+begin_src emacs-lisp
+(use-package org-roam
+  :init
+  (require 'org-ref)
+  :config
+  ;;...
+  )
+#+end_src
+
+*org-cite*
+
+Org-cite should be loaded by default, and no additional setup should be
+required.
+
+** Using the Cached Information
+
+It is common to use take reference notes for academic papers. To designate the
+node to be the canonical node for the academic paper, we can use its unique
+citation key:
+
+#+begin_src org
+,* Probabilistic Robotics
+:PROPERTIES:
+:ID:       51b7b82c-bbb4-4822-875a-ed548cffda10
+:ROAM_REFS: @thrun2005probabilistic
```

Comment:
I don't really have anything meaningful to add on this, but AFAIU this is not something that would be actionable (interactively) by org? Since this is a custom syntax. 

IIRC org-ref's "cite:" were actionable, just like URL links. I'm not sure whether this even important, but just wanted to highlight this, so someone else who's more knowledgeable on citations would evaluate this.

Reply:
I raised this question on the org-mode list recently, though have not yet had any responses.

https://lists.gnu.org/archive/html/emacs-orgmode/2021-08/msg00317.html

My tentative suggestion there was to allow citations in the property drawers, but I am still not sure.

As @Wetlize guesses, references of this form won't be treated as citations by org, and so they don't be actionable.

Thoughts?

Reply:
Just to update, org-cite dev says it's a "terrible idea" to allow cites in property drawers :-)

https://lists.gnu.org/archive/html/emacs-orgmode/2021-09/msg00130.html